src/icons/Cross.tsx

```diff
@@ -0,0 +1,30 @@
+export default function () {
+  return (
+    <svg
+      width="18"
+      height="17"
+      viewBox="0 0 18 17"
```

Comment:
Can you make it `0 0 18 18`. It should be rectangle (you may try to export it again from figma, if needed)